example-application/api.js

```diff
@@ -35,15 +35,15 @@ const initializeWebServer = async (customMiddleware) => {
   });
 };
 
-const stopWebServer = async () => {
+const stopWebServer = () => {
```

Comment:
Why not async?

Reply:
nothing async in this function

Reply:
Maybe I miss something but connection.close is calling you back when it's closed, this is asynchronous by nature. We wrap it with promise to allow sane await syntax

How would you do await stopWebServer() if it is not async?

Reply:
The only reason to set function as `async` is if you `await` something in it.

Reply:
@mikicho Compilation-wise you're right, but there is also the convention of tagging async functions for the convenience of the user who does not have to dig inside to refer the return type. Not critical, we can leave this way

Reply:
Not sure I understand, VScode automatically require `await` for this function with or without the `async` (because its return a `Promise`)